File: Prince_Tiwari_Assignment1_VB/MirrorImageTriangle.vb
Public Class MirrorImageTriangle
    Public Shared Sub Main(args As String())
        Dim n As Integer = Convert.ToInt32(Console.ReadLine())

        For i As Integer = 1 To n
            For j As Integer = 1 To i - 1
                Console.Write(" ")
            Next j
            For k As Integer = i To n
                Console.Write(k & " ")
            Next k
            Console.WriteLine()
        Next i

        For i As Integer = n To 2 Step -1
            For j As Integer = 2 To i - 1
                Console.Write(" ")
            Next j
            For k As Integer = i - 1 To n
                Console.Write(k & " ")
            Next k
            Console.WriteLine()
        Next i
    End Sub
End Class

File: Prince_Tiwari_Assignment1/Mirror_Image_Triangle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Prince_Tiwari_Assignment1
{
    internal class Mirror_Image_Triangle
    {
        public static void Main(string[] args)
        {
            int n = Convert.ToInt32(Console.ReadLine());

            for (int i = 1; i <= n; i++)
            {
                for (int j = 1; j < i; j++)
                {
                    Console.Write(" ");
                }
                for (int k = i; k <= n; k++)
                {
                    Console.Write(k + " ");
                }
                Console.WriteLine();
            }

            for (int i = n; i > 1; i--)
            {
                for (int j = 2; j < i; j++)
                {
                    Console.Write(" ");
                }
                for (int k = i - 1; k <= n; k++)
                {
                    Console.Write(k + " ");
                }
                Console.WriteLine();
            }
        }
    }
}